src/app/shared/clarin-shared-util.ts

```diff
@@ -76,3 +76,9 @@ export function loadItemAuthors(item, itemAuthors, baseUrl) {
   });
   itemAuthors.next(itemAuthorsLocal);
 }
+
+export function makeLinks(text: string): string {
```

Comment:
isn't there a library for this?

Reply:
Yes, there is a library `linkify`, I used it and committed changes.

Reply:
There is a `linkify` library, but it is deprecated and cannot work seamlessly with our DSpace Angular 13. Although I successfully imported that library, it broke 227 tests, indicating that I would need to update approximately 100 test files. Therefore, I prefer to use our local `makeLinks` function instead of `linkify`.